From c10dfa3e6288e7847d8cfec7cb49f793c4ba3412 Mon Sep 17 00:00:00 2001
From: John Greene <jogreene@redhat.com>
Date: Wed, 10 Sep 2014 18:29:10 -0400
Subject: [ethernet] ixgbe: fix message terminations

Message-id: <1410373750-11341-51-git-send-email-jogreene@redhat.com>
Patchwork-id: 93548
O-Subject: [RHEL7.1 PATCH 50/50] ixgbe: e1000e/igb/ixgbe/i40e: fix message terminations
Bugzilla: 1091123
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>

BZ: 1091123
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7919168
Tested: In beaker. See BZ for details

[redhat] filtered for ixgbe only

Upsream: c5ffe7e1f745984b37b8ffe03b03f3d716a072f3

Add \n at the end of messages where missing, remove all \r.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/ixgbe/ixgbe_common.c b/drivers/net/ethernet/intel/ixgbe/ixgbe_common.c
index 24fba39..981b8a7 100644
--- a/drivers/net/ethernet/intel/ixgbe/ixgbe_common.c
+++ b/drivers/net/ethernet/intel/ixgbe/ixgbe_common.c
@@ -1195,7 +1195,7 @@ static s32 ixgbe_detect_eeprom_page_size_generic(struct ixgbe_hw *hw,
   */
  hw->eeprom.word_page_size = IXGBE_EEPROM_PAGE_SIZE_MAX - data[0];
 
- hw_dbg(hw, "Detected EEPROM page size = %d words.",
+ hw_dbg(hw, "Detected EEPROM page size = %d words.\n",
         hw->eeprom.word_page_size);
 out:
  return status;
diff --git a/drivers/net/ethernet/intel/ixgbe/ixgbe_phy.c b/drivers/net/ethernet/intel/ixgbe/ixgbe_phy.c
index 6c70db3..c73a239 100644
--- a/drivers/net/ethernet/intel/ixgbe/ixgbe_phy.c
+++ b/drivers/net/ethernet/intel/ixgbe/ixgbe_phy.c
@@ -535,7 +535,7 @@ s32 ixgbe_setup_phy_link_generic(struct ixgbe_hw *hw)
 
  if (time_out == max_time_out) {
   status = IXGBE_ERR_LINK_SETUP;
-  hw_dbg(hw, "ixgbe_setup_phy_link_generic: time out");
+  hw_dbg(hw, "ixgbe_setup_phy_link_generic: time out\n");
  }
 
  return status;
@@ -744,7 +744,7 @@ s32 ixgbe_setup_phy_link_tnx(struct ixgbe_hw *hw)
 
  if (time_out == max_time_out) {
   status = IXGBE_ERR_LINK_SETUP;
-  hw_dbg(hw, "ixgbe_setup_phy_link_tnx: time out");
+  hw_dbg(hw, "ixgbe_setup_phy_link_tnx: time out\n");
  }
 
  return status;
@@ -1174,7 +1174,7 @@ s32 ixgbe_identify_sfp_module_generic(struct ixgbe_hw *hw)
     status = 0;
    } else {
     if (hw->allow_unsupported_sfp) {
-     e_warn(drv, "WARNING: Intel (R) Network Connections are quality tested using Intel (R) Ethernet Optics.  Using untested modules is not supported and may cause unstable operation or damage to the module or the adapter.  Intel Corporation is not responsible for any harm caused by using untested modules.");
+     e_warn(drv, "WARNING: Intel (R) Network Connections are quality tested using Intel (R) Ethernet Optics.  Using untested modules is not supported and may cause unstable operation or damage to the module or the adapter.  Intel Corporation is not responsible for any harm caused by using untested modules.\n");
      status = 0;
     } else {
      hw_dbg(hw,
diff --git a/drivers/net/ethernet/intel/ixgbe/ixgbe_ptp.c b/drivers/net/ethernet/intel/ixgbe/ixgbe_ptp.c
index c247a22..8902ae6 100644
--- a/drivers/net/ethernet/intel/ixgbe/ixgbe_ptp.c
+++ b/drivers/net/ethernet/intel/ixgbe/ixgbe_ptp.c
@@ -456,7 +456,7 @@ void ixgbe_ptp_rx_hang(struct ixgbe_adapter *adapter)
   IXGBE_READ_REG(hw, IXGBE_RXSTMPH);
   adapter->last_rx_ptp_check = jiffies;
 
-  e_warn(drv, "clearing RX Timestamp hang");
+  e_warn(drv, "clearing RX Timestamp hang\n");
  }
 }
 
@@ -512,7 +512,7 @@ static void ixgbe_ptp_tx_hwtstamp_work(struct work_struct *work)
   dev_kfree_skb_any(adapter->ptp_tx_skb);
   adapter->ptp_tx_skb = NULL;
   clear_bit_unlock(__IXGBE_PTP_TX_IN_PROGRESS, &adapter->state);
-  e_warn(drv, "clearing Tx Timestamp hang");
+  e_warn(drv, "clearing Tx Timestamp hang\n");
   return;
  }
 
-- 
1.7.1